.travis.yml

```diff
@@ -1,16 +1,13 @@
 language: ruby
 dist: trusty
+sudo: required
 rvm:
 - 2.4.4
 before_script:
   - gem install bundler -v 1.16.2
   - bundle exec rake hatchet:setup_travis
-addons:
-  apt:
-    sources:
-    - heroku
-    packages:
-    - heroku-toolbelt
+before_install:
+  - sudo bash etc/ci-setup.sh
```

Comment:
I know i keep my scripts in `etc/` but it's really the wrong place strictly speaking (`etc/` is for config files as i understand it). in theory, they should go in `bin/` the way the [Go buildpack does it](https://github.com/heroku/heroku-buildpack-go/tree/master/bin). I never liked cluttering `bin/` though, and I like keeping only buildpack API scripts in there. A better place for this script is maybe `opt/`. idk. i've never cared enough to move mine. So i'll leave it to you.

Reply:
This is good to know 👍 I'll refactor to that if/when I convert to Heroku CI - the wrong place is good enough for now to get some other PRs unblocked.

Reply:
👍